.github/workflows/ci.yaml

```diff
@@ -790,6 +790,31 @@ jobs:
           node-version: ${{ env.NODEJS_VERSION }}
       - uses: actions/checkout@v3.5.2
 
+      - id: yarn-cache
+        name: Restore Yarn Cache
+        uses: actions/cache@v3.3.1
+        with:
+          key: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
+          path: ./.yarn/
+          restore-keys: |
+            ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
+      - run: ./tools/ci.sh
+  cactus-plugin-ledger-connector-polkadot:
+    continue-on-error: false
+    env:
+      FULL_BUILD_DISABLED: true
+      JEST_TEST_PATTERN: packages/cactus-plugin-ledger-connector-polkadot/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts
+      JEST_TEST_RUNNER_DISABLED: false
+      TAPE_TEST_RUNNER_DISABLED: true
+    needs: build-dev
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Use Node.js ${{ env.NODEJS_VERSION }}
+        uses: actions/setup-node@v3.6.0
+        with:
+          node-version: ${{ env.NODEJS_VERSION }}
+      - uses: actions/checkout@v3.5.2
+
```

Comment:
@AnmolBansalDEV Could you please apply the latest patterns here with path filtering conditional execution? It's some fairly simple boilerplate syntax that you can mostly copy from existing packages. Please see the main branch's ci.yaml file for examples and let me know if there are any questions.

Reply:
@petermetz updated

Reply:
@AnmolBansalDEV Thank you. There's a part of it that you missed (actually using the change detection's output) so I posted a suggestion for it in my review. It's a very simple change but without it the change detection does not work unfortunately so we need it as well.

Reply:
updated

Reply:
> updated

@AnmolBansalDEV Thank you!